### PR TITLE
Adapt to vsc-install

### PR DESCRIPTION
**Pull request by stdweird**

### CODE REVIEW

File: bin/mympisanity.py [hunk 1]
```diff
@@ -135,15 +139,20 @@ def check():
 if __name__ == '__main__':
     log = getLogger('mympisanity')
     setLogLevelInfo()
+
+    if MPI4PY_EXCEPTION:
+        log.error("No mpi4py found: %s", MPI4PY_EXCEPTION)
+        sys.exit(1)
```
Comment (wpoely86) [inline bin/mympisanity.py]: didn't we switch to raising exception upon fatal errors?

Comment (stdweird) [inline bin/mympisanity.py]: this is the main of a script, why would you want to raise something here?

File: lib/vsc/mympirun/rm/__init__.py [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
-##
+# along with vsc-mympirun.  If not, see <http://www.gnu.org/licenses/>.
+#
+"""
+Allow other packages to extend this namespace, zip safe setuptools style
+"""
+import pkg_resources
+pkg_resources.declare_namespace(__name__)
 
-import local, pbs
```
Comment (wpoely86) [inline lib/vsc/mympirun/rm/__init__.py]: shouldn't these be moved?

Comment (stdweird) [inline lib/vsc/mympirun/rm/__init__.py]: this will probably break the current code, yes. but the `__init__` should be empty, it's up to jeroen to fix this

**Comment by stdweird** [comment]
@JensTimmerman this is now with subclassable shared_setup
@wpoely86 @lepoev: requires new vsc-install in pypi

**Comment by JensTimmerman** [comment]
why do you need to override this vsc.setup inner class, 
can't you just pass

```
setup(
    ...
      cmdclass={'install': mympirun_vsc_install_scripts }
     ...
)
```
(or in the PACKAGE) ?

or does this not work with the overridden inner class things?

**Comment by JensTimmerman** [comment]
oh, it says it in the comments, ` +# because easy_install ignores the easy_install cmdclass `
ok then :+1:

**Comment by stdweird** [comment]
as `vsc_setup` is a class, it seems odd not to do modifications via subclassing. the main reason the original mokey patch code failed is that the entry in the SHARED_SETUP cmdclass was not updated. the `_fvs` trickery in `parse_target` does this now.

**Comment by stdweird** [comment]
test this please

**Comment by stdweird** [comment]
@Iepoev: cherry-pick https://github.com/hpcugent/vsc-mympirun/pull/52/commits/94eb27cd764facaee548fde13bce55760e2cc684 and do a `easy_install --user vsc-install`, should be fixed now

**Comment by JensTimmerman** [comment]
`oh, # because easy_install ignores the easy_install cmdclass`
wel, actually it seems it doesn't have to ignore this anymore, but we choose to delete these unknown command classes in vsc-install, not easy_install ignoring this, vsc-install.

**Pull request merged**